mnefun/_mnefun.py

```diff
@@ -1368,10 +1369,10 @@ def do_preprocessing_combined(p, subjects):
             meg, eeg = 'meg' in raw, 'eeg' in raw
             picks = pick_types(raw.info, meg=meg, eeg=eeg, eog=False,
                                exclude=[])
-            assert type(p.auto_bad_reject) and type(p.auto_bad_flat) == dict
+            flat = p.auto_bad if type(p.auto_bad_flat) == dict else None
```

Comment:
The following line does epoching using p.auto_bad and p.auto_bad reject to get to the `badchs` via channels scores from the `Epochs` drop log. I am trying to give users the option to use `p.auto_bad` with or without rejection of trials where channels are flat i.e., `p.auto_bad_flat` in case e.g. when MF sets channels to zero after cHPI failure.